---
# Documentation: https://wowchemy.com/docs/managing-content/

title: "Long-read metagenome-assembled genomes improve identification of novel complete biosynthetic gene clusters in a complex microbial activated sludge ecosystem"
subtitle: ''
summary: ''
authors:
- Matin Nuhamunada
tags:
- 'genome mining'
- 'natural products'
- 'metagenome'
categories: []
date: '2022-11-29'
lastmod: ''
featured: false
draft: false

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ''
  focal_point: ''
  preview_only: false

# Projects (optional).
#   Associate this post with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["internal-project"]` references `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects: []
publishDate: '2021-01-15T21:34:29.167017Z'
publication_types:
- '2'
abstract: 'Microorganisms produce a wide variety of secondary/specialized  metabolites (SMs), the majority of which are yet to be  discovered. These natural products play multiple roles in  microbiomes and are important for microbial competition,  communication, and success in the environment. SMs have been our  major source of antibiotics and are used in a range of  biotechnological applications. In silico mining for biosynthetic  gene clusters (BGCs) encoding the production of SMs is commonly  used to assess the genetic potential of organisms. However, as  BGCs span tens to over 200 kb, identifying complete BGCs requires  genome data that has minimal assembly gaps within the BGCs, a  prerequisite that was previously only met by individually  sequenced genomes. Here, we assess the performance of the  currently available genome mining platform antiSMASH on 1,080  high-quality metagenome-assembled bacterial genomes (HQ MAGs)  previously produced from wastewater treatment plants (WWTPs)  using a combination of long-read (Oxford Nanopore) and short-read  (Illumina) sequencing technologies. More than 4,200 different  BGCs were identified, with 88\% of these being complete. Sequence  similarity clustering of the BGCs implies that the majority of  this biosynthetic potential likely encodes novel compounds, and  few BGCs are shared between genera. We identify BGCs in abundant  and functionally relevant genera in WWTPs, suggesting a role of  secondary metabolism in this ecosystem. We find that the assembly  of HQ MAGs using long-read sequencing is vital to explore the  genetic potential for SM production among the uncultured members  of microbial communities.'
publication: '*American Society of Microbiology*'
url_pdf: https://journals.asm.org/doi/reader/10.1128/msystems.00632-22
doi: 10.1128/msystems.00632-22
---
