---
title: Long-read metagenome-assembled genomes improve identification of novel complete
  biosynthetic gene clusters in a complex microbial activated sludge ecosystem
authors:
- Roberto SÃ¡nchez-Navarro
- Matin Nuhamunada
- Omkar S Mohite
- Kenneth Wasmund
- Mads Albertsen
- Lone Gram
- Per H Nielsen
- Tilmann Weber
- Caitlin M Singleton
date: '2022-12-01'
publishDate: '2024-11-27T20:04:27.585529Z'
publication_types:
- article-journal
publication: '*mSystems*'
abstract: Microorganisms produce a wide variety of secondary/specialized metabolites
  (SMs), the majority of which are yet to be discovered. These natural products play
  multiple roles in microbiomes and are important for microbial competition, communication,
  and success in the environment. SMs have been our major source of antibiotics and
  are used in a range of biotechnological applications. In silico mining for biosynthetic
  gene clusters (BGCs) encoding the production of SMs is commonly used to assess the
  genetic potential of organisms. However, as BGCs span tens to over 200 kb, identifying
  complete BGCs requires genome data that has minimal assembly gaps within the BGCs,
  a prerequisite that was previously only met by individually sequenced genomes. Here,
  we assess the performance of the currently available genome mining platform antiSMASH
  on 1,080 high-quality metagenome-assembled bacterial genomes (HQ MAGs) previously
  produced from wastewater treatment plants (WWTPs) using a combination of long-read
  (Oxford Nanopore) and short-read (Illumina) sequencing technologies. More than 4,200
  different BGCs were identified, with 88% of these being complete. Sequence similarity
  clustering of the BGCs implies that the majority of this biosynthetic potential
  likely encodes novel compounds, and few BGCs are shared between genera. We identify
  BGCs in abundant and functionally relevant genera in WWTPs, suggesting a role of
  secondary metabolism in this ecosystem. We find that the assembly of HQ MAGs using
  long-read sequencing is vital to explore the genetic potential for SM production
  among the uncultured members of microbial communities. IMPORTANCE Cataloguing secondary
  metabolite (SM) potential using genome mining of metagenomic data has become the
  method of choice in bioprospecting for novel compounds. However, accurate biosynthetic
  gene cluster (BGC) detection requires unfragmented genomic assemblies, which have
  been technically difficult to obtain from metagenomes until very recently with new
  long-read technologies. Here, we determined the biosynthetic potential of activated
  sludge (AS), the microbial community used in resource recovery and wastewater treatment,
  by mining high-quality metagenome-assembled genomes generated from long-read data.
  We found over 4,000 BGCs, including BGCs in abundant process-critical bacteria,
  with no similarity to the BGCs of characterized products. We show how long-read
  MAGs are required to confidently assemble complete BGCs, and we determined that
  the AS BGCs from different studies have very little overlap, suggesting that AS
  is a rich source of biosynthetic potential and new bioactive compounds.
tags:
- activated sludge; biosynthetic gene cluster; metagenome-assembled genome; secondary
  metabolite; wastewater treatment plant
---
